### PR TITLE
Improve deployment to ease verification and documentation

### PR DESCRIPTION
**Pull request by izqui**
Builds on #422

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,19 +1,16 @@
...
 
 #### 👋 We are tagging tasks that are [good first issues](https://github.com/aragon/aragonOS/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) so you can get started contributing to Aragon Core.
 Don't be shy to contribute even the smallest tweak. Everyone will be especially nice and helpful to beginners to help you get started!
 
 ## Documentation
 
-Visit the [Aragon wiki](https://wiki.aragon.one/dev/aragonOS_index/) for in depth documentation on the [architecture](https://wiki.aragon.one/documentation/aragonOS/) and different parts of the system.
+Visit the [Aragon Developer Portal](https://hack.aragon.org/docs/aragonos-intro.html) for in depth documentation on the [architecture](https://hack.aragon.org/docs/aragonos-ref.html) and different parts of the system.
 
 ## Installing aragonOS
```
Comment (sohkai) [inline readme.md]: While we're at it, I think we should change the heading of this too, to be something like "Get started contributing" under the "Contributing" section at the bottom.

Installing `aragonOS` is literally `npm install`.

File: scripts/deploy-apm.js [hunk 2]
```diff
@@ -99,7 +106,10 @@ module.exports = async (
 
   log('=========')
   const apmAddr = receipt.logs.filter(l => l.event == 'DeployAPM')[0].args.apm
-  log('Deployed APM:', apmAddr)
+  log('# APM:')
+  log('Address:', apmAddr)
+  log('Transaction hash:', receipt.tx)
+  log('=========')
   log(apmAddr)
```
Comment (sohkai) [inline scripts/deploy-apm.js]: Probably don't need this trailing bit anymore.

File: scripts/helpers/create-logger.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
+const flatten = require('truffle-flattener')
```
Comment (sohkai) [inline scripts/helpers/create-logger.js]: Seems like this file would be better named as "deploy-logger"

File: scripts/helpers/create-logger.js [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+const mkdirp = require('mkdirp')
+const fs = require('fs')
+const path = require('path')
+
+const truffleConfig = require('../../truffle-config')
```
Comment (sohkai) [inline scripts/helpers/create-logger.js]: Using `path.resolve()` would be better

File: scripts/deploy-apm.js [hunk 5]
```diff
@@ -76,12 +77,18 @@ module.exports = async (
...
+    apmRegistryBase.address,
+    apmRepoBase.address,
+    ensSubdomainRegistrarBase.address,
+    ensAddress,
+    '0x00'
+  )
+  await logDeploy(apmFactory, { verbose })
 
   log(`Assigning ENS name (${labelName}.${tldName}) to factory...`)
   try {
```
Comment (sohkai) [inline scripts/deploy-apm.js]: Tried `truffle exec --network rpc scripts/deploy-apm.js` by itself and hit the ENS error here.

Maybe we should also set `aragonpm.eth` if we deploy our own instance of ENS in this script?

File: scripts/deploy-daofactory.js [hunk 6]
```diff
@@ -10,31 +12,30 @@ module.exports = async (
...
+    await logDeploy(evmScriptRegistryFactory, { verbose })
   }
   const daoFactory = await DAOFactory.new(
     kernelBase.address,
     aclBase.address,
     evmScriptRegistryFactory ? evmScriptRegistryFactory.address : ZERO_ADDR
   )
 
-  log('DAOFactory deployed:', daoFactory.address)
+  logDeploy(daoFactory, { verbose })
```
Comment (sohkai) [inline scripts/deploy-daofactory.js]: I think this should have an `await`?

File: scripts/deploy-apm.js (hunk 5 above)
Comment (izqui) [reply scripts/deploy-apm.js]: It blows up if you don't pass the first account as the `OWNER` env variable, fixing it.

Comment (izqui) [reply scripts/deploy-apm.js]: Added this in 04813a1:

```
...
OWNER env variable not found, setting APM owner to the provider\'s first account
Owner: 0xb4124ceb3451635dacedd11767f004d8a28c6ee7
...
```

If you run `truffle exec --network rpc scripts/deploy-apm.js` now it should just work.

**Pull request merged**